src/main/resources/templates/LogIn.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<title>Online HTML Editor</title>
```

Comment:
Same as before, these belong within the head tag

Reply:
Will discuss with Nolan